pkg/scanner/local/scan.go

```diff
@@ -309,16 +309,10 @@ func toDetectedMisconfiguration(res ftypes.MisconfResult, defaultSeverity dbType
 	}
 
 	var primaryURL string
-	if strings.HasPrefix(res.Namespace, "builtin.") {
+
+	if res.ID != "" && res.ID != "N/A" {
```

Comment:
What if the rule is defined by user? The link will be 404?

Reply:
If the rule is custom the ID will be empty, so we will not set a link at all.

Reply:
ID can be defined in custom policies, no?
https://aquasecurity.github.io/trivy/v0.27.1/docs/misconfiguration/custom/#rego-format

Reply:
That's why we used to check namespace. It is fine if we no longer need it.

Reply:
I fixed it so it checks if there is no namespace (defsec go rule) or the namespace is `builtin` (built-in rego rule) we generate a link, otherwise we leave it blank.

Reply:
Thanks for updating!